tutorials/Guides_for_HPC_clusters/Longleaf

```diff
@@ -0,0 +1,115 @@
+# Longleaf
+
+Longleaf is a Linux-based computing system available for free to researchers across UNC Chapel Hill's campus. Workloads where each job requires a single compute host are best suited for this cluster.
+
+> :bulb: This guide should be used alongside tutorial 02 ([here](https://github.com/jacksund/simmate/blob/main/tutorials/02_%20Run_a_workflow.md#switching-to-a-remote-cluster)).
+<br/>
+
+# The Simmate checklist
+
+From the posted requirements:
+
+1. a VASP license for your team **(COMPLETED)**
+2. a remote cluster that you have a profile with 
+3. VASP installed on the remote cluster **(COMPLETED)**
+4. Anaconda installed on the remote cluster **(COMPLETED)**
+5. Simmate installed on the remote cluster
+6. VASP Potentials in `~/simmate/vasp/Potentials` on the remote cluster. Feel free to reach out with Scott with questions (scw@email.unc.edu)
+
+For the uncompleted requirements:
+
+2. you must create a profile on longleaf with your onyen. Contact UNC ITS to enquire about making a profile.
+5. make your personal environment named `yourname_env` (e.g. `jacks_env`)
+6. follow tutorial 02 ([here](https://github.com/jacksund/simmate/blob/main/tutorials/02_%20Run_a_workflow.md#configuring-potentials-for-vasp-users)) to set up VASP potentials
+
+<br/>
+
+# Example commands and scripts
+
+Substitute these commands with the ones you see in the main tutorial.
+
+<br/>
+
+## Sign in with...
+```
+ssh youronyen@longleaf.unc.edu
+```
+
+<br/>
+
+## Load VASP with...
+```
+module vasp/5.4.4
+```
+
+<br/>
+
+## Call VASP with... 
+```
+mpirun vasp_std
+```
+
+<br/>
+
+## Create your conda env with...
+```
+conda create -n my_env -c conda-forge python=3.8 simmate
+conda activate my_env
+# and initialize your database for the first time
+simmate database reset
+```
+
+<br/>
+
+## Access scratch directory with...
+```
+cd /pine/scr/y/o/youronyen
+```
+
+<br/>
+
+## Create a SLURM script with...
+```
+nano submit.sh
+```
+
+<br/>
+
+And paste in the example content...
+```
+#! /bin/sh
+
+#SBATCH -J holder
+#SBATCH -n 20
+#SBATCH -N 1
+#SBATCH --mem=96g
+#SBATCH -p general
+#SBATCH -o slurm.out
+#SBATCH --mail-type=FAIL
+#SBATCH --mail-user=youronyen@live.unc.edu
+#SBATCH -t 11-00:00
+
+simmate workflows run energy-mit POSCAR -c "mpirun vasp_std > vasp.out"
+```
```

Comment:
Can you update this using the submit.sh I use for Warwulf? Couple of things to note here:

- using verbose names is better for beginners (e.g. use `--time` instead of `-t`)
- 96GB is overkill for most jobs. So that we don't get in trouble and so our jobs start faster, I would drop this to 40GB
- The mpirun command needs updated (see my other review comment)